### PR TITLE
RTL FIXES

### PR DESCRIPTION
**Pull request by Anush2303**
## Related Issue(s)

- Fixes #

### CODE REVIEW

File: packages/react-components/react-charts-preview/stories/src/VerticalBarChart/VerticalBarChartDefault.stories.tsx [hunk 1]
```diff
@@ -242,12 +242,12 @@ export const VCBasic = () => {
             lineLegendText={'just line'}
             lineLegendColor={'brown'}
             lineOptions={lineOptions}
-            {...(isCalloutselected && {
+            /*{...(isCalloutselected && {
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/stories/src/VerticalBarChart/VerticalBarChartDefault.stories.tsx:245]: why disabled this

File: packages/react-components/react-charts-preview/stories/src/VerticalBarChart/VerticalBarChartStyled.stories.tsx [hunk 2]
```diff
@@ -41,21 +41,19 @@ export const VCStyled = () => {
...
-          barWidth={20}
-          useSingleColor={useSingleColor}
-          yAxisTickCount={6}
-          colors={customColors}
-          lineLegendColor={`rgb(174, 140, 0)`}
-          hideLegend={true}
-          enableReflow={true}
-        />
-      </div>
+      <VerticalBarChart
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/stories/src/VerticalBarChart/VerticalBarChartStyled.stories.tsx:44]: why removed div container

File: packages/react-components/react-charts-preview/stories/src/VerticalBarChart/VerticalBarChartDefault.stories.tsx (hunk 1 above)
Comment (Anush2303) [reply packages/react-components/react-charts-preview/stories/src/VerticalBarChart/VerticalBarChartDefault.stories.tsx:245]: Custom callout is not working. Needs to be fixed.

File: packages/react-components/react-charts-preview/stories/src/VerticalBarChart/VerticalBarChartStyled.stories.tsx (hunk 2 above)
Comment (Anush2303) [reply packages/react-components/react-charts-preview/stories/src/VerticalBarChart/VerticalBarChartStyled.stories.tsx:44]: Will revert this. Probably had been trying something

**Review by AtishayMsft — APPROVED** [review]

File: packages/react-components/react-charts-preview/stories/src/VerticalBarChart/VerticalBarChartDefault.stories.tsx (hunk 1 above)
Comment (Anush2303) [reply packages/react-components/react-charts-preview/stories/src/VerticalBarChart/VerticalBarChartDefault.stories.tsx:245]: Okay

**Pull request merged**